src/transactions/transactions.service.ts

```diff
@@ -87,7 +87,7 @@ export class TransactionsService {
         );
       }
     } catch (error) {
-      handleSdkError(error);
+      handleSdkError(error as Error);
```

Comment:
```suggestion
      handleSdkError(error);
```

This shouldn't be needed and doesn't provide any additional type safety

Reply:
Seems like something changed that makes us need it now with new packages.

OK to leave